terraform/workspaces/infra/s3/s3.tf

```diff
@@ -67,6 +67,22 @@ resource "aws_s3_bucket" "cdn" {
   }
 }
 
+resource "aws_s3_bucket" "logs" {
+  count = var.disable_logs ? 0 : 1
+
+  bucket        = "${var.workspace}-logs"
+  acl           = "private"
```

Comment:
Having `acl` and `server_side_encryption_configuration` inside the `aws_s3_bucket` is deprecated. These should be split into their own resources along with blocking public access. See [examples here](https://github.com/useparagon/paragon/blob/release/next/terraform/workspaces/ci/cache/s3.tf).

Reply:
Updated. PTAL